internal/daemon/controller/handlers/authmethods/password_test.go

```diff
@@ -627,7 +627,7 @@ func TestAuthenticate_Password(t *testing.T) {
 	}
 }
 
-func TestAuthenticate_AuthAccountConnectedToIamUser_Password(t *testing.T) {
+func TestAuthenticate_AuthAccountConnectedToIamUsername_Password(t *testing.T) {
```

Comment:
Hmm I am not sure this should change, perhaps it should be `username` but this is outside of the scope of credentials and thus this PR. I do get it is just a test name so feel free to keep it changed if you like